gapic/packaging/dependencies.yaml

```diff
@@ -49,7 +49,11 @@ gax_version:
   ruby:
     lower: '0.8.0'
   java:
-    lower: '0.6.1'
+    lower: '1.2.0'
+
+gax_grpc_version:
```

Comment:
ahhh okay I have reverted the whole file and add the change later once my code PR is complete.